tasks/sqlcmd_input_file.yml

```diff
@@ -42,11 +42,15 @@
         or
         (__mssql_sqlcmd_input is failed)
       block:
-        - name: >-
-            Print the output of the sqlcmd command
-            if not empty for {{ item }}
+        - name: Print the output from inputting {{ __mssql_sqlcmd_loop }}
```

Comment:
why not just
```yaml
        - name: Print the output from inputting {{ item }}
          debug:
            msg: |
              STDOUT
              {{ __mssql_sqlcmd_input.stdout }}
              STDERR
              {{ __mssql_sqlcmd_input.stderr }}
```
?

Reply:
It doesn't print the output as pretty as `debug: var:`, I now simplified the whole thing and removed a redundant block. The loop is still there though, with it the output and the filename are ### printed separately, which is easier to read.